api.json

```diff
@@ -1311,6 +1312,22 @@
          "mempool_coins": {
            "type":"boolean",
            "description":"Any Rosetta implementation that can update an AccountIdentifier's unspent coins based on the contents of the mempool should populate this field as true. If false, requests to `/account/coins` that set `include_mempool` as true will be automatically rejected."
+          },
+         "block_hash_case": {
+           "description":"This specifies the normalized case for block hash in the BlockIdentifier. If not specified, it's assumed to be case sensitive.",
+           "type":"string",
+           "nullable": true,
+           "enum": [
+             "upper_case",
+             "lower_case",
+             "case_sensitive",
+              null
+            ],
+           "default":"case_sensitive"
+          },
+         "transaction_hash_case": {
+           "$ref":"#/components/schemas/Allow/properties/block_hash_case",
```

Comment:
Why does `transaction_hash_case` refer to `block_hash_case` - Is it just for saying that it has same properties as `block_hash_case` and just the `description` is different?

Reply:
https://swagger.io/docs/specification/using-ref/ - does this section `$ref and Sibling Elements` applies here? If yes - both hash case will be same 🤔

Reply:
I think this is a quirk of openapi and case.yaml was inlined to block_hash_case